docs/source/conf.py

```diff
@@ -27,7 +27,7 @@
 # The short X.Y version
 version = ""
 # The full version, including alpha/beta/rc tags
-release = u'4.7.0'
+release = "4.9.1"
```

Comment:
Should this be 4.9.2?
```suggestion
release = "4.9.2"
```

Reply:
Already?